internal/coremsgs/en_config_descriptions.go

```diff
@@ -20,301 +20,265 @@ import (
+	ConfigBlockchainFabricFabconnectChannel      = ffc("config.blockchain.fabric.fabconnect.channel", "The Fabric channel that FireFly will use for BatchPin transactions", stringType)
+	ConfigBlockchainFabricFabconnectPrefixLong   = ffc("config.blockchain.fabric.fabconnect.prefixLong", "The prefix that will be used for Fabconnect specific HTTP headers when FireFly makes requests to Fabconnect", stringType)
+	ConfigBlockchainFabricFabconnectPrefixShort  = ffc("config.blockchain.fabric.fabconnect.prefixShort", "The prefix that will be used for Fabconnect specific query parameters when FireFly makes requests to Fabconnect", stringType)
+	ConfigBlockchainFabricFabconnectSigner       = ffc("config.blockchain.fabric.fabconnect.signer", "The Fabric signing key to use when submitting transactions to Fabconnect", stringType)
+	ConfigBlockchainFabricFabconnectTopic        = ffc("config.blockchain.fabric.fabconnect.topic", "The websocket listen topic that the node should register on, which is important if there are multiple nodes using a single Fabconnect", stringType)
+	ConfigBlockchainFabricFabconnectURL          = ffc("config.blockchain.fabric.fabconnect.url", "The URL of the Fabconnect instance", "URL "+stringType)
+
+	ConfigBlockchainFabricFabconnectProxyURL = ffc("config.blockchain.fabric.fabconnect.proxy.url", "The URL for the Fabconnect proxy", "URL "+stringType)
+
+	ConfigBroadcastBatchAgentTimeout = ffc("config.broadcast.batch.agentTimeout", "How long to keep around a batching agent for a sending identity before disposal", stringType)
+	ConfigBroadcastBatchPayloadLimit = ffc("config.broadcast.batch.payloadLimit", "The maximum payload size of a batch for broadcast messages", byteSizeType)
+	ConfigBroadcastBatchSize         = ffc("config.broadcast.batch.size", "The maximum number of messages that can be packed into a batch", intType)
+	ConfigBroadcastBatchTimeout      = ffc("config.broadcast.batch.timeout", "The timeout to wait for a batch to fill, before sending", timeDurationType)
+
+	ConfigCorsCredentials = ffc("config.cors.credentials", "CORS setting to control whether a browser allows credentials to be sent to this API", booleanType)
+	ConfigCorsDebug       = ffc("config.cors.debug", "Whether debug is enabled for the CORS implementation", booleanType)
+	ConfigCorsEnabled     = ffc("config.cors.enabled", "Whether cors is enabled", booleanType)
+	ConfigCorsHeaders     = ffc("config.cors.headers", "CORS setting to control the allowed headers", stringType)
+	ConfigCorsMaxAge      = ffc("config.cors.maxAge", "The maximum age a browser should rely on CORS checks", timeDurationType)
+	ConfigCorsMethods     = ffc("config.cors.methods", " CORS setting to control the allowed methods", stringType)
+	ConfigCorsOrigins     = ffc("config.cors.origins", "CORS setting to control the allowed origins", stringType)
+
+	ConfigDatabaseMaxChartRows = ffc("config.database.maxChartRows", "The maximum rows to fetch for each histogram bucket", intType)
```

Comment:
I assert this config key is wrong (not related to your change).
`config.database.XYZ` should be reserved for plugins of name `XYZ`.
This should be changed to `config.histograms.maxChartRows` or similar

Reply:
Makes sense. Want me to change it in this PR?

Reply:
Thanks for moving that 👍